layouts/_default/baseof.html

```diff
@@ -6,7 +6,7 @@
     {{ partial "head.html" . }}
   </head>
   <body class="td-{{ .Kind }}{{ with .Page.Params.body_class }} {{ . }}{{ end }}">
-    <a rel="me" href="https://fosstodon.org/@opentelemetry"></a>
+    <a rel="me" href="https://fosstodon.org/@opentelemetry" aria-label="fosstodon.org/@opentelemetry"></a>
```

Comment:
If we must add an ARIA attribut, then this element should be declared as [hidden](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden):

```suggestion
    <a rel="me" href="https://fosstodon.org/@opentelemetry" aria-hidden="true"></a>
```

Reply:
Done thanks